SmartDeviceLink/private/SDLSoftButtonManager.m

```diff
@@ -125,7 +125,7 @@ - (void)setSoftButtonObjects:(NSArray<SDLSoftButtonObject *> *)softButtonObjects
     // Set the soft button ids. Check to make sure no two soft buttons have the same name, there aren't many soft buttons, so n^2 isn't going to be bad
     for (NSUInteger i = 0; i < softButtonObjects.count; i++) {
         NSString *buttonName = softButtonObjects[i].name;
-        softButtonObjects[i].buttonId = i * 100;
+        softButtonObjects[i].buttonId = (i * 100) ? (i * 100) : (i + 1);
```

Comment:
```suggestion
       // HAX: Due to a SYNC 3.0 bug (https://github.com/smartdevicelink/sdl_ios/issues/1793#issue-708356008), a `buttonId` can not be zero. As a workaround we will start the `buttonId`s from 1.
        softButtonObjects[i].buttonId = (i * 100) ? (i * 100) : (i + 1);
```

Reply:
Also, If we start the `buttonId`s from `1` we will have to update the "ok" `buttonId` in the Swift and Obj-C example apps alert. If you click on the right most button in both example apps you will see that the alert keeps reappearing. This is due to the "ok" `buttonId` being set to `1`.

Reply:
@FrankElias77 This still needs to be fixed